Subject: skip part of test-crypto-engine to avoid rebuilding
 For now, only tests not requiring built artifacts are run
Forwarded: not-needed
Author: Jérémy Lal <kapouer@melix.org>
Last-Update: 2022-05-02
--- a/test/parallel/test-crypto-engine.js
+++ b/test/parallel/test-crypto-engine.js
@@ -34,7 +34,7 @@
   if (common.isOSX)
     engineLib = `lib${engineName}.dylib`;
   else if (common.isLinux && process.arch === 'x64')
-    engineLib = `lib${engineName}.so`;
+    //engineLib = `lib${engineName}.so`;
 
   if (engineLib !== undefined) {
     const execDir = path.dirname(process.execPath);
